sjb/config/test_cases/ci-kubernetes-conformance-node-e2e-containerized-rhel.yml

```diff
@@ -0,0 +1,59 @@
+---
+parent: 'common/test_cases/kubernetes.yml'
+overrides:
+  junit_analysis: False
+  timer: "H */2 * * *"
+extensions:
+  sync_repos:
+    - name: "aos-cd-jobs"
+  actions:
+    - type: "script"
+      title: "Install go 1.9.1"
```

Comment:
We should:

1. get this repo to have 1.9 https://cbs.centos.org/repos/paas7-openshift-origin39-candidate/x86_64/os/Packages/
2. use:
    ```yaml
    - type: "host_script"
      title: "install Go 1.9"
      script: |-
        oct prepare golang --version=1.9.1 -u https://cbs.centos.org/repos/paas7-openshift-origin36-candidate/x86_64/os/
    ```

Reply:
That requires the ``oct`` to be installed on the provisioned AWS instance. Where can I pick it?

Reply:
No, it should require `oct` on the control host (`"host_script"`)

Reply:
Oh, it will install the golang remotely. Ok.

Reply:
Even with ``https://cbs.centos.org/repos/paas7-openshift-origin39-candidate/x86_64/os/`` you get at most golang-1.8.3.

Reply:
Even CentOS Koji has not 1.9 golang build yet: https://cbs.centos.org/koji/packageinfo?packageID=16

Reply:
EEK